src/tiny.ts

```diff
@@ -39,34 +39,41 @@ export class TinyPg {
          port: parseInt(params.port, 10),
          database: params.pathname.split('/')[1],
          ssl: params.query.sslmode !== 'disable',
+         connectionTimeoutMillis: pool_options.connection_timeout_ms,
```

Comment:
might as well spell `connectionTimeoutMilliseconds`

Reply:
The name is designated by the pgpool options type.